packages/react/src/components/field-label/field-label.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  FieldLabel,
+  FieldLabelProps,
+  HelperText,
+  Input,
+  Textarea,
+} from '@onfido/castor-react';
+import React from 'react';
+import { Meta, omit, Story } from '../../../../../docs';
+
+export default {
+  title: 'React/FieldLabel',
+  component: FieldLabel,
+  argTypes: {
+    ...omit<FieldLabelProps>('className', 'style'),
+    children: { control: 'text' },
+  },
+  args: {
+    children: 'Label',
+  },
+} as Meta<FieldLabelProps>;
+
+export const Playground: Story<FieldLabelProps> = (props: FieldLabelProps) => (
+  <FieldLabel {...props} />
+);
+
+interface FieldLabelWithHelperTextProps extends FieldLabelProps {
+  label: string;
+  helperText: string;
+}
+
+export const WithHelperText = ({
+  label,
+  helperText,
+  ...restFieldLabelProps
+}: FieldLabelWithHelperTextProps) => (
+  <FieldLabel {...restFieldLabelProps}>
+    {label}
+    <HelperText>{helperText}</HelperText>
+  </FieldLabel>
+);
+WithHelperText.argTypes = omit<FieldLabelWithHelperTextProps>('children');
+WithHelperText.args = {
+  label: 'Label',
+  helperText: 'Helper text',
+};
+
+interface FieldLabelWithInputProps extends FieldLabelProps {
+  id: string;
+  label: string;
+}
+
+export const WithInput = ({
+  id,
+  label,
+  ...restFieldLabelProps
+}: FieldLabelWithInputProps) => (
+  <>
+    <FieldLabel {...restFieldLabelProps} htmlFor={id}>
+      {label}
+    </FieldLabel>
+    <Input id={id} />
```

Comment:
I feel we should encourage nesting the input inside the label, but that's subjective.

```tsx
<FieldLabel {...restFieldLabelProps} htmlFor={id}>
  {label}
  <Input id={id} />
</FieldLabel>
```

What do you think?

Reply:
Probably you have noticed that both Input/Textarea and now FieldLabel stories have different usage examples. I wanted to show variations of how these can be composed.

Whether they should be nested or not - I'm not too sure what's more appropriate. But with this I wanted to show that thet can be separate but _must_ be connected via the ID. For example, if you'd to have a table with two columns one which hold the label, then other holds an input.

Do you strongly suggest that these should be nested? I don't mind changing.

Reply:
No, that makes sense. Showing different options is good too.